### PR TITLE
17319 - Date Requirement

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:* /bcgov/[entity17319](https://github.com/bcgov/entity/issues/17319)

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-527-z4e1jea7.web.app

File: src/components/SpecialResolution/Resolution.vue [hunk 1]
```diff
@@ -257,6 +260,10 @@ export default class Resolution extends Vue {
   async updateIsEditingSpecialResolution (val): Promise<void> {
     await this.setEditingSpecialResolution(val)
   }
+
+  onResolutionDate (date: string) {
+    this.changedResolutionDate = date
+  }
```
Comment (severinbeauvais) [inline src/components/SpecialResolution/Resolution.vue:266]: This is fine but next time I'd be tempted to just put this code on line 133.

(In this case, one less function is one less thing to look at when trying to understand this code.)

File: src/components/SpecialResolution/ResolutionEditor.vue [hunk 2]
```diff
@@ -244,11 +244,7 @@ export default class ResolutionEditor extends Vue {
 
   /** The maximum resolution date that can be entered (today). */
   get resolutionDateMax (): string {
-    if (this.getSpecialResolution.signingDate) {
-      return DateUtilities.dateToYyyyMmDd(DateUtilities.yyyyMmDdToDate(this.getSpecialResolution.signingDate))
-    } else {
-      return this.getCurrentDate
-    }
+    return this.getCurrentDate
```
Comment (severinbeauvais) [inline src/components/SpecialResolution/ResolutionEditor.vue:247]: 👍 

Even though this computed function is trivial, it's alongside resolutionDateMin(), so there's a nice symmetry / consistency when I look for what the code does.

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by JazzarKarim — APPROVED** [review]
LGTM, great job Jia 👍

File: src/components/SpecialResolution/Resolution.vue (hunk 1 above)
Comment (Jxio) [reply src/components/SpecialResolution/Resolution.vue:266]: ok will do it next time, thanks!

**Pull request merged**